### PR TITLE
Results list

### LINKED ISSUE #31: Create ResultsList Component

### PR DESCRIPTION
**Pull request by bretm9**
### Participating Group Members:

- @bretm9 

### Code Highlights:

- Added `v-for` in `ResultsList` to render `ResultListItems` from the `searchResults` data.
- Added basic HTML for `ResultsListItem` to test that the list is properly rendering `ResultsListItems`
- Added initial basic testing structure

### Have Tests Been Added?

- [ ] No.
- [ ] Yes, but all tests are not passing.
- [x] Yes, and all are passing.

### Any background context you want to provide?
N/A

### Message/Questions for reviewer:
N/A

### Issues:

- Addresses #33 
- Closes #31 

### Screenshots (if appropriate):
Screen Shot 2021-01-08 at 9 57 33 AM

### Tracking Consistency:

- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] All new and existing tests passed
- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

**Review by npdarrington — APPROVED** [review]
Looks good!

**Review by nicolegooden — APPROVED** [review]
Awesome! Looks great.  Stacy, feel free to rebase and merge when you're ready!

File: src/components/ResultsList.vue [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+export default {
+  props: ['searchResults'],
+  components: { ResultsListItem }
+}
+</script>
+
+<style>
+  .results-list {
+    margin: auto;
+    width: 70%;
```
Comment (stacyp2006) [inline src/components/ResultsList.vue:21]: We already talked about this on another pr, but just a reminder to add the scss attribute to the style tag on your next branch.

**Review by stacyp2006 — APPROVED** [review]
Looks good! Merging now...

**Pull request merged**